### PR TITLE
DOCSP-41464-dump-export-mongodb-only-notice

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Adding disclaimer to mongodump, mongorestore, mongoimport, mongoexport, mongotop, mongostat, and mongofiles pages that MongoDB Database tools binaries are not supported for non - MongoDB deployments.
- Added monospace to the title page on ``mongodump`` for consistency. 
- **Note**: Language based loosely on the [mongosh considerations page](https://www.mongodb.com/docs/compass/current/connect/#considerations). We did not want to call out specific competitor products in the announcement and used the term `non-genuine MongoDB deployment`.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongodump/

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongorestore/

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongoimport/

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongoexport/

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongostat/

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongofiles/

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-41464/mongotop/

## JIRA

https://jira.mongodb.org/browse/DOCSP-41464

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66a3eafe26eba79552783fbd

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jwilson-mdb — APPROVED** [review]
Thanks @ianf-mongodb, LGTM!

File: snooty.toml [hunk 1]
```diff
@@ -73,3 +73,19 @@ section = 1
...
+        "mongorestore.txt",
+        "mongoimport.txt",
+        "mongoexport.txt",
+        "mongotop.txt",
+        "mongostat.txt",
+        "mongofiles.txt"
+        ]
+variant = "warning"
+value = """\
+        MongoDB Command Line Database Tool binaries are not supported or tested for use with non-genuine MongoDB deployments. Any usage of these tools with non-genuine MongoDB deployments may result in data corruption.
```
Comment (mvankeulen94) [inline snooty.toml:90]: Some of these tools don't modify data. Maybe instead we can say:
"Data integrity is not guaranteed when using these tools with non-genuine MongoDB deployments."

Comment (mvankeulen94) [reply snooty.toml:90]: Thinking further, maybe we can borrow from the existing language around compatibility with EOL'd versions:
"MongoDB Command Line Database Tool binaries are not supported or tested for use with non-genuine MongoDB deployments. While the tools may work on these deployments, any such compatibility is not guaranteed."

Comment (ianf-mongodb) [reply snooty.toml]: Sure thing I updated to the latest suggestion.

**Review by ianf-mongodb — COMMENTED** [review]
Updated @mvankeulen94

File: snooty.toml (hunk 1 above)
Comment (mvankeulen94) [reply snooty.toml]: Thank you! The text looks great - I believe Abhi has shared the message about this change so let's give it a bit longer for them to respond before we push.

**Review by mvankeulen94 — APPROVED** [review]
Lgtm

**Pull request merged**